src/fiatconnect/saga.ts

```diff
@@ -1002,13 +1003,25 @@ export function* handleCreateFiatConnectTransfer(
   const quoteId = fiatConnectQuote.getQuoteId()
   let transactionHash: string | null = null
   try {
-    const { transferAddress }: TransferResponse = yield call(_initiateTransferWithProvider, action)
+    const { transferAddress, transferId }: TransferResponse = yield call(
+      _initiateTransferWithProvider,
+      action
+    )
 
     if (flow === CICOFlow.CashOut) {
       transactionHash = yield call(_initiateSendTxToProvider, {
         transferAddress,
         fiatConnectQuote,
       })
+      yield put(
+        cacheFiatConnectTransfer({
+          txHash: transactionHash!,
```

Comment:
I'm aware we are trying not to use the non-null assertion operator. However, the return type from the generator is currently `any`, so we're already casting to make the transactionHash into a string.